lld/ELF/Relocations.cpp

```diff
@@ -1714,7 +1714,8 @@ void elf::postScanRelocations() {
         if (!sym.isDefined()) {
           replaceWithDefined(sym, *in.plt,
                              target->pltHeaderSize +
-                                 target->pltEntrySize * sym.getPltIdx(),
+                                 (uint64_t)target->pltEntrySize *
+                                     sym.getPltIdx(),
```

Comment:
What does this change have to do with the change in the COFF linker?

Reply:
You are right, the changes in ELF are unrelated to COFF changes. 
Put them in the same PR because they are from the same report.

Reply:
1468946 [Unintentional integer overflow](https://scan6.scan.coverity.com/doc/en/cov_checker_ref.html#static_checker_OVERFLOW_BEFORE_WIDEN)
The expression's value may not be what the programmer intended, because the expression is evaluated using a narrow (i.e. few bits) integer type.

In lld::​elf::​postScanRelocations()::​[lambda(lld::​elf::​Symbol &) (instance 1)]::​operator ()(lld::​elf::​Symbol &): An integer overflow occurs, with the result converted to a wider integer type ([CWE-190](http://cwe.mitre.org/data/definitions/190.html))

Reply:
This seems an unnecessary change. PLT entries are expected to small, much smaller than 2GiB.
If we really want to do something, we could add a check `PltSection::addEntry`, but that feels quite unnecessary.